products/sle/main.pm

```diff
@@ -1182,6 +1182,12 @@ elsif (get_var("RESCUESYSTEM")) {
     loadtest "installation/rescuesystem_validate_sle";
 }
 elsif (get_var("INSTALLCHECK")) {
+    if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
+        loadtest "installation/bootloader_hyperv";
+    }
+    else {
+        loadtest "installation/bootloader_svirt";
+    }
```

Comment:
Hm, when other backends already work without a specific bootloader test module before "rescuesystem" how about implementing bootloader helper functions or using existing ones from lib/bootloader_setup.pm and using them where necessary within installation/rescuesystem as in tests/boot/boot_do_desktop.pm ?

Reply:
I untangled VM setup and mangling with GRUB/sys-inst, so running "unusual" scenarios on svirt needs to call just one sub.